cvat/schema.yml

```diff
@@ -1531,6 +1531,64 @@ paths:
       responses:
         '204':
           description: The invitation has been deleted
+  /api/invitations/{key}/accept:
+    post:
+      operationId: invitations_create_accept
+      parameters:
+      - in: path
+        name: key
+        schema:
+          type: string
+        description: A unique value identifying this invitation.
+        required: true
+      tags:
+      - invitations
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InvitationWriteRequest'
+        required: true
+      security:
+      - {}
+      responses:
+        '200':
+          content:
+            application/vnd.cvat+json:
+              schema:
+                $ref: '#/components/schemas/InvitationWrite'
```

Comment:
I'm not sure why this needs to return a response body, but even if it does, shouldn't it be `InvitationRead` rather than `InvitationWrite`?

Reply:
Fixed